installation/resources/tiller.yaml

```diff
@@ -125,11 +125,9 @@ spec:
               namespace: kube-system
             type: Opaque
             EOF
-            )
 
-            echo "${COMBO_YAML}" | kubectl apply -f -
+            timeout 35s bash -c 'until kubectl create -f /tmp/secret.yaml ; do sleep 5; done'
```

Comment:
will this work with upgrade? kubectl create returns error if object already exists